java/midp/com/sun/midp/l10n/LocalizedStringsBase.java

```diff
@@ -1,3 +1,29 @@
+/*
```

Comment:
OK. So, in the future, whenever we add a file from the reference implementation we need to add a license header if it isn't already present?